AnalysisScripts/CondAdLIF_FIWeightMap.m

```diff
@@ -49,20 +50,20 @@
 PopParams.p0spike = 0.15;
 
 %%
-TimeParams.dt      = 0.05;
+TimeParams.dt      = 5;
```

Comment:
hey @jmg1030 - just a heads up, TimeParams.dt is still the simulation dt, you want to keep this small for accurate simulations.  The dt you want to change (i.e. the output dt) is an optional input to the EMAdLif function: 'save_dt',5